### DISCUSSION TITLE
Self hosting for only 10 people in the company

### DISCUSSION THREAD

**Discussion by Variety7838** [outline/outline#6629] [Self Hosting] [answered]
Hi,
I'm not that expert about the licencing stuff. Our company doesn't use these kind of documentation software but only our team wants to use something. It's about 8-10 people at most and we don't need support nor help in the setup. Could you please help me what's the minimum plan that we could choose? We also doesn't need anything else, only the limited features that's shown [here](https://www.getoutline.com/pricing). But I think we must choose the business plan because we're a company. Is it the only option?
Thank you for your help!

**Comment by tommoor** [comment, marked as answer]
You can self-host for free, just follow the documentation:
https://docs.getoutline.com/s/hosting/doc/docker-7pfeLP5a8t

If you need support or the extra features at any point you can purchase a license.

**Reply by putradimas** [reply]
Is it okay to use it in production? e.g. using the docker image and deploying it as a production knowledge base for our internal team.

My current understanding is that as long as we are deploying it for internal company use, we can use it in production. Is this correct?

**Reply by tommoor** [reply]
That's correct, but there is no support of course other than these forums.
